modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -10,12 +10,15 @@ class PowerOfAttorney::RequestController < ClaimsApi::V2::Veterans::PowerOfAttor
         FORM_NUMBER = 'POA_REQUEST'
 
         def request_representative
-          target_veteran
-          @claims_api_forms_validation_errors = validate_form_2122_and_2122a_submission_values(user_profile)
+          poa_code = form_attributes.dig('poa', 'poaCode')
+          @claims_api_forms_validation_errors = validate_form_2122_and_2122a_submission_values(
+            target_veteran.participant_id, user_profile, poa_code
+          )
```

Comment:
_Technically_, we are altering the behavior of the `/power-of-attorney-request` endpoint (beyond the scope of this task). However, the change is behind a feature flag, and we will need to do the validation here eventually. So might as well update here now rather than invent a workaround to later undo 🤷